### PR TITLE
[TACHYON-1428] Documented parts of the tests of the tachyon.client package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java [hunk 1]
```diff
@@ -20,14 +20,23 @@
 
 import tachyon.conf.TachyonConf;
 
+/**
+ * Tests for the {@link GetLineageInfoListOptions} class.
+ */
 public class GetLineageInfoListOptionsTest {
 
+  /**
+   * Tests that the option can successfully be build.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java]: build -> built

File: clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java [hunk 2]
```diff
@@ -20,14 +20,23 @@
...
+   */
   @Test
   public void buildTest() {
     Assert.assertNotNull("The GetLineageInfoListOptions should not be null",
         new GetLineageInfoListOptions.Builder(new TachyonConf()).build());
   }
 
+  /**
+   * Tests that the default for this option can successfully be build.
```
Comment (yupeng9) [inline clients/unshaded/src/test/java/tachyon/client/lineage/options/GetLineageInfoListOptionsTest.java]: build -> built

**Comment by yupeng9** [comment]
Thanks for the improvement. I left some minor comments.

**Comment by HorizonNet** [comment]
@yupeng9 Thanks for the comments. I updated the PR accordingly.

**Comment by HorizonNet** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**